autogen_subdomain_delegation.go

```diff
@@ -0,0 +1,61 @@
+package api
+
+import (
+	"encoding/json"
+	"gopkg.in/nullstone-io/go-api-client.v0/types"
+	"net/http"
+	"path"
+	"strconv"
+)
+
+type AutogenSubdomainDelegation struct {
+	Client *Client
+}
+
+// GET /orgs/:orgName/subdomains/:subdomainId/envs/:envName/autogen_subdomain_delegation
+func (d AutogenSubdomainDelegation) Get(subdomainId int, envName string) (*types.AutogenSubdomain, error) {
```

Comment:
Looks like this is returning an `autogen_subdomain`.
If that's the case, is there value having a `GET` endpoint and `Get` function if you can get it from `GET .../autogen_subdomains` endpoint

Reply:
I have this here to satisfy the terraform provider. Should I route the terraform provider for autogen_subdomain_delegation to the autogen_subdomain endpoint?

Reply:
I removed the delegation#show endpoint and instead nested the delegation under the autogen_subdomain.